src/languageProvider/locales/en_US.json

```diff
@@ -101,6 +101,10 @@
   "leaderboard.mostQTUM": "Who bet the most QTUM",
   "leaderboard.ranking": "Ranking",
   "leaderboard.title": "Leaderboard",
+  "leaderboard.totalBOT": "Total bet in BOT",
```

Comment:
I think `Total arbitrated in BOT` or `Total voted in BOT` makes more sense? Since voting with BOT isn't a bet.